force-app/main/default/objects/Deal_Support_Process__c/recordTypes/Enterprise_Advisory.recordType-meta.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<RecordType xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Enterprise_Advisory</fullName>
+    <active>true</active>
+    <label>Enterprise Advisory</label>
```

Comment:
actually Advisory is alone in NOT having Enterprise in front of it.

Reply:
The API name and label are now both restored to their original values, "Advisory".